izpack-maven-plugin/src/main/java/org/izpack/mojo/IzPackNewMojo.java

```diff
@@ -164,6 +164,18 @@ public class IzPackNewMojo extends AbstractMojo
     @Parameter
     private String excludeProperties;
 
+    /**
+     * List of properties to be included.
+     */
+    @Parameter
+    private String[] includeProperties;
```

Comment:
You can change the type as `Set<String>` instead of `String[]`, you don't need code to add these values to `Set<String> includeFilter`.